lib/cpp/win_delay_load_hook.cc

```diff
@@ -0,0 +1,25 @@
+#ifdef _MSC_VER

```

Comment:
I found this file that seems to be more complete but I do not really understand what it does 😛